src/Tizen.NUI/src/public/BaseComponents/TextEditorBindableProperty.cs

```diff
@@ -607,6 +607,41 @@ public partial class TextEditor
             Tizen.NUI.Object.GetProperty((System.Runtime.InteropServices.HandleRef)textEditor.SwigCPtr, TextEditor.Property.InputUnderline).Get(out temp);
             return temp;
         }));
+
+         /// This will be public opened in tizen_7.0 after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
If Strikethrough and InputStrikethrough are not provided as properties, BindableProperty is also not required.

Reply:
@wonrst , What I have to do here? should I remove both BindableProperty Strikethrough and InputStrikethroug ?
For example Underline has BindableProperty, should not we do the same thing for Strikethrough?!

Reply:
Yes, you should remove both BindableProperty Strikethrough and InputStrikethrough?
The reason `Underline` has a BindableProperty is because the property `PropertyMap Underline` exists.

But we do not plan to provide properties of Strikethrough and InputStrikethrough.
So BindableProperty is also unnecessary.

Reply:
It would be good to refer to SetInputFilter()/GetInputFilter().

Reply:
Okay, thank you

Reply:
Done